consul/base.py

```diff
@@ -640,13 +646,38 @@ def delete(self, key, recurse=None, cas=None, token=None, dc=None):
             return self.agent.http.delete(
                 CB.json(), '/v1/kv/%s' % key, params=params)
 
+    class TXN(object):
```

Comment:
`txn` is just following the docs here: https://www.consul.io/api/txn.html. Looking at Hashicorp's Go client, it's also using `txn`.